src/components/inputs/ui-select-extended/ui-select-extended.tsx

```diff
@@ -69,6 +69,16 @@ const UISelectExtended: React.FC<OHRIFormFieldProps> = ({ question, handler, onC
     question.value = handler?.handleFieldSubmission(question, value, encounterContext);
   };
 
+  useEffect(() => {
+    if (!isEmpty(previousValue)) {
```

Comment:
Is this a control we want to add a previous value to? So much is happening already with the search and might have some points of failure or conflict

Reply:
I see, although `previousValue` was already supported by `ui-select-extended` prior to this PR.